runtime/pack/dist/opt/termdebug/plugin/termdebug.vim

```diff
@@ -717,7 +717,7 @@ def ContinueCommand()
 enddef
 
 # This is global so that a user can create their mappings with this.
-def TermDebugSendCommand(cmd: string)
+def g:TermDebugSendCommand(cmd: string)
```

Comment:
if it worked, then you may need to update the doc/help as well, there is some description about this func.

Reply:
From my script, I can `call TermDebugSendCommand("...")` without the leading `g:`, but if the function is not declared with the `g:` then the call fails with `E117: Unknown function: TermDebugSendCommand`. I'm not sure the doc needs to be updated.